content/en/docs/setup/install/helm/index.md

```diff
@@ -166,6 +166,15 @@ installed above.
     $ kubectl delete namespace istio-system
     {{< /text >}}
 
+## Uninstall stable revision label resources
+
+If you decide to continue using the old control plane, instead of completing the upgrade,
```

Comment:
> you can uninstall the newer revision using

nit: this doesn't uninstall the revision, just the tag?

Reply:
Another nit, we should say "revision tag" instead of label?

Reply:
The header for all instances refers to `revision labels` in both the canary and helm portions. This was not changed. Updated the docs to reference the tag only. The user must then perform an uninstall.